packages/manager/modules/sign-up/src/components/siret/siret.controller.js

```diff
@@ -29,6 +31,11 @@ export default class SiretCtrl {
     );
 
     this.trackingPrefix = TRACKING_PREFIX[this.trackingMode];
+    this.vatProperty = get(
+      VAT_SUBSIDIARIES_FIELD_VALUE,
+      this.country.toUpperCase(),
+      'vat',
+    );
```

Comment:
```suggestion
    this.vatProperty = VAT_SUBSIDIARIES_FIELD_VALUE[this.country.toUpperCase()] || 'vat';
```

Reply:
Ok, it's more logical but I saw in the codebase that lodash.get is more used than Object['key'] so I thought loadash.get was the preferred way. I will change